.ci/update-tekton-bundle.sh

```diff
@@ -0,0 +1,79 @@
+APPSTUDIO_TASKS_REPO=quay.io/repository/redhat-appstudio-qe/test-images
+PIPELINE_BUNDLE_IMG=quay.io/repository/redhat-appstudio-qe/test-images:pipeline-bundle-$JVM_BUILD_SERVICE_PR_SHA
+BUILD_DEFINITIONS_DIR="$WORKSPACE"/tmp/build-definitions
+
+function cloneBuildDefinitions() {
+    git clone https://$GITHUB_TOKEN@github.com/redhat-appstudio/build-definitions.git "$BUILD_DEFINITIONS_DIR"
+}
+
+function updateAnalyzerImage() {
+    yq e -i "(.[] \
+    | select(.path == \"/spec/steps/-\").value \
+    | select(.name == \"analyse-dependencies-java-sbom\").image) \
+    |= \"$JVM_BUILD_SERVICE_ANALYZER_IMAGE\"" \
+    $BUILD_DEFINITIONS_DIR/tasks/patches/add-sbom-and-push.yaml
+}
+
+function createAndPushTaskBundle() {
+    if [[ $(uname) = Darwin ]]; then
+        CSPLIT_CMD="gcsplit"
+    else
+        CSPLIT_CMD="csplit"
+    fi
+
+    PART=1
+    COUNT=0
+    TASK_TEMP=$(mktemp -d)
+    PIPELINE_TEMP=$(mktemp -d)
+    oc kustomize $BUILD_DEFINITIONS_DIR/tasks | $CSPLIT_CMD -s -f $TASK_TEMP/task -b %02d.yaml - /^---$/ '{*}'
+    oc kustomize $BUILD_DEFINITIONS_DIR/pipelines/base > ${PIPELINE_TEMP}/base.yaml
+
+    ## Limit number of tasks in bundle
+    MAX=10
+    TASKS=
+    REF="$APPSTUDIO_TASKS_REPO:task-bundle-$JVM_BUILD_SERVICE_PR_SHA-$PART"
+    for TASK in $TASK_TEMP/task*.yaml; do
+        TASK_NAME=$(yq e ".metadata.name" $TASK)
+        if [ $COUNT -eq $MAX ]; then
+            echo Creating $REF
+            tkn bundle push $TASKS $REF
+            COUNT=0
+            PART=$((PART+1))
+            TASKS=
+        fi
+        REF="$APPSTUDIO_TASKS_REPO:task-bundle-$JVM_BUILD_SERVICE_PR_SHA-$PART"
+
+        # Replace appstudio-utils placeholder by newly build appstudio-utils image
+        #yq e -i ".spec.steps[] |= select(.image == \"appstudio-utils\").image=\"$APPSTUDIO_UTILS_IMG\"" $TASK
+
+        TASKS="$TASKS -f $TASK"
+        for file in $PIPELINE_TEMP/*.yaml; do
+            yq e -i "(.spec.tasks[].taskRef | select(.name == \"$TASK_NAME\")) |= {\"name\": \"$TASK_NAME\", \"bundle\":\"$REF\"}" $file
+            yq e -i "(.spec.finally[].taskRef | select(.name == \"$TASK_NAME\")) |= {\"name\": \"$TASK_NAME\", \"bundle\":\"$REF\"}" $file
+        done
+        COUNT=$((COUNT+1))
+    done
+    echo Creating $APPSTUDIO_TASKS_REPO:task-bundle-$JVM_BUILD_SERVICE_PR_SHA-$PART
+    tkn bundle push $TASKS $REF
+}
+
+function createAndPushPipelineBundles() {
+    tkn bundle push $PIPELINE_BUNDLE_IMG -f ${PIPELINE_TEMP}/base.yaml
+}
+
+function createConfigMap() {
+    oc apply -m build-templates -f - << CONFIGMAP
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: build-pipelines-defaults
```

Comment:
I think you'll need to specify a well known namespace that is accessible like `default` or `openshift`, or perhaps create a new one, so that when the e2e's follow their typical pattern of creating a test namespace, our jvm build service e2e code can find this config map and copy it into the test namespace.

Reply:
I created it in this namespace because it would override all pipeline runs from all components (that is, unless a specific component had a configmap like that created in the same namespace it lives, then it would take precedence over the one in build-pipelines-defaults).

But if you think it's best to have it copied to a specific namespace as needed, we can go with that approach as well. :)

Reply:
ah I was unaware of had forgotten that element of the config map override features you had added @brunoapimentel thanks for the clarification.  I'm fine with the global override.

That said, you have a typo in the line above.  I'll add a suggestion comment